server/project.go

```diff
@@ -480,6 +493,27 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusBadRequest, "Schema path template is supported only if branch doesn't have wildcard.")
 		}
 
+		// Make sure the branch exists in the repo.
```

Comment:
Is there a way to share the code with the above?

Reply:
I put the duplicate code into a function.